app/views/landing_page/blocks/_heading.html.erb

```diff
@@ -0,0 +1,10 @@
+<%
+  heading_level = block.data["heading_level"] || 2
+  padding = block.data["padding"] || false
```

Comment:
If the setting is no longer in the YAML, and I assume that means that the padding should always be "true", then this line can be removed too.

Reply:
@leenagupte Whoops, Monday morning brain - have removed now!